src/pages/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,15 @@
+import { Footer, Header } from "../../../components";
+import { useDocumentTitle } from "../../../customHooks/useDocumentTitle";
+import "./Login.css";
+
+export const Login = () => {
+	useDocumentTitle("Login | MS");
+
```

Comment:
Can you explain why you are using this.

Reply:
I have created a useEffect custom hook to set the title of the page. So when the user will go tp login page, the title will be changed to 'Login | MS"